.github/workflows/spotless_checks.yml

```diff
@@ -0,0 +1,17 @@
+name: Spotless Checks
+
+on:
+  schedule:
+    - cron: "0 5 * * *"
+
+jobs:
+  build:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Spotless Apply
+        run: ./gradlew spotlessCheck
```

Comment:
This action does not contribute anything useful at the moment because it neither raises a PR with the updated formatting, nor pushes the result of the checks somewhere. Does `spotlessCheck` allow us to generate and upload a report of some sort?

Reply:
I am marking this PR as WIP for now. I will see if I can add the PR creation as well